tests/profiles/test_models.py

```diff
@@ -0,0 +1,12 @@
+from .test_base import BaseTest
+from authors.apps.profiles.models import Profile
+
+
+class TestProfile(BaseTest):
+
+    def test_return_profile(self):
+        self.client.post('/api/users/', self.registration_data, format='json')
+        self.client.get('/api/profiles/user/', format='json')
+        profile = Profile.objects.latest('id')
+        print(profile)
```

Comment:
@NLSanyu print statements are not supposed to be pushed. Please remove it.

Reply:
Thanks @PatrickCmd  for the feedback. I'm working on it.

Reply:
@PatrickCmd I have finished working on all your feedback